packages/xo-web/src/common/xo-compression/index.js

```diff
@@ -0,0 +1,7 @@
+export SelectCompression from './select-compression'
+
+// zstd detection source: https://github.com/xapi-project/xen-api/commit/adca3b750da06deb9a141ad2447605601c49a177
+export const isZstdSupported = container =>
+  container.type === 'host'
+    ? container.license_params.restrict_zstd_export === 'false'
+    : container.restrictions.restrict_zstd_export === 'false'
```

Comment:
Should we expose this in `xapi-object-to-xo.js`?

Reply:
I think is a good idea :+1: